languages/rust/exercises/concept/options/tests/options.rs

```diff
@@ -2,85 +2,84 @@ use options::*;
 
 #[test]
 fn test_reviving_dead_player() {
-    let level: u32 = 34;
     let dead_player = Player {
         health: 0,
         mana: Some(0),
-        level,
+        level: 34,
     };
-    let revived_player = dead_player.revive().expect("reviving a dead player must return Some(player)");
+    let revived_player = dead_player
+        .revive()
+        .expect("reviving a dead player must return Some(player)");
     assert_eq!(revived_player.health, 100);
     assert_eq!(revived_player.mana, Some(100));
-    assert_eq!(revived_player.level, level);
+    assert_eq!(revived_player.level, dead_player.level);
 }
 
 #[test]
 #[ignore]
 fn test_reviving_alive_player() {
-    let level: u32 = 8;
     let alive_player = Player {
         health: 1,
         mana: None,
-        level,
+        level: 8,
     };
-    assert_eq!(alive_player.revive().is_none(), true);
+    assert_eq!(alive_player.revive(), None);
```

Comment:
Oh I figured out why it was this way. If you `assert_eq` with `None`, then the student would have to make the Player PartialEq and Debug. Okay, in that case I'm down with:

```suggestion
    assert!(alive_player.revive().is_none());
```